### DISCUSSION TITLE
V380E11_WF6_AKQ8 support?

### DISCUSSION THREAD

**Discussion by dliebchen** [Arkady23/V380-Web#46] [General]
Hey there,
very good work as far as I can see :) 

Any chance to add my camera model?
V380 App says:
AppV380E11_V2.7.0.7_20220106
HwV380E11_WF6_AKQ8_MS32006_20180628
Link: https://www.banggood.com/de/GUUDGO-10LED-5X-Zoom-HD-2MP-IP-Security-Camera-WiFi-Wireless-1080P-Outdoor-PTZ-Waterproof-Night-Vision-ONVIF-p-1527199.html?cur_warehouse=CZ&ID=47184
(actually it's not 1080p...)

I tried nearly all of the patches I could find in your repository but none of them seem to work. Maybe I'm doing something wrong but as far as I can see I only have to copy those files/folders (ark-add-on / updatepatch / local_update.conf / patch_reuse) onto a FAT32 formatted mSD-card. Nothing happens after powering on the cam and a portscan tells me there are still the same ports open (554 (rtsp) / 8800 (app communication?) / 8899 (onvif soap) / 9800 (don't know)).

I also tried some wireshark magic but couldnt find any update lookup in the logs during updatecheck on my phone.

Thanks for your awesome work :) I run some china cams with my object-detection setup and most of them can be configured through a webinterface. Sadly the V380 based cams don't have such interface and writing configuration with onvif commands doesnt work somehow... (disable timestamp overlay, setting framerate and some other stuff)

Best Regards :)

**Comment by Arkady23** [comment]
Added a description for installing a web server manually for new cameras.

**Comment by Shinchan1215** [comment]
Hi did it work I have same model

**Comment by Arkady23** [comment]
A new way to install a webcam has appeared. I'll describe it on the main page soon.
